HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -45,8 +45,8 @@ def self.calculate(weather, hpxml_bldg, cfa, hvac_systems)
       apply_hvac_loads(hvac_heating, hvac_sizing_values, system_design_loads, ducts_heat_load, ducts_cool_load_sens, ducts_cool_load_lat)
       apply_hvac_size_limits(hvac_cooling)
       apply_hvac_heat_pump_logic(hvac_sizing_values, hvac_cooling)
-      apply_hvac_equipment_adjustments(hvac_sizing_values, weather, hvac_heating, hvac_cooling, hvac_system)
-      apply_hvac_installation_quality(hvac_sizing_values, hvac_heating, hvac_cooling)
+      apply_hvac_equipment_adjustments(hvac_sizing_values, weather, hvac_heating, hvac_cooling, hvac_system) # FIXME: this is hvac_sizing_values.Heat_Airflow = xxx, etc.
+      apply_hvac_installation_quality(hvac_sizing_values, hvac_heating, hvac_cooling) # FIXME: airflow cfm override should occur after this?
```

Comment:
> Need help/guidance on where to implement the airflow cfm override.
> 
> @jonwinkler @yzhou601

Just confirm, you're talking about the maximum allowed airflow rate, right? If so, I think the answer depends on if a fixed capacity input should take priority over the max airflow input. Meaning, what if the fixed capacity input results in an airflow greater than the maximum allowed, should the fixed capacity be reduced? Or should the maximum allowed airflow rate be ignored?

Reply:
Yes, I'm talking about the maximum allowed airflow rate.

Reply:
Okay. Any thoughts on whether the max airflow rate or fixed capacity should take priority?

Reply:
It's a good question. Seems like the fixed capacity input should be used when provided. (Otherwise if the resulting airflow is greater than the maximum allowed, we'd need to find/calculate the fixed capacity such that the airflow rate _equals_ the maximum allowed?)

I suppose this means the maximum allowed airflow rate, when also provided, should be either be ignored or should throw an error. Probably OK to ignore it. This means that the maximum allowed airflow rate is _only_ used when defaulting the capacity to an autosized value?

Reply:
Good points. I agree. Maybe we name it maximum_autosized_airflow to make it clear how it's used. 

I'll take another look at hvac_sizing.rb to see where this fits.

Reply:
Hmm, that said, it does look like autosized airflow rates get adjusted (proportionally) when fixed capacity is provided. So perhaps this is somewhat analogous to that? Meaning, we could (proportionally) adjust fixed capacity for a "fixed" (i.e., potentially adjusted) airflow rate?

Maybe that would mean this fits directly following the `apply_hvac_fixed_capacities` method? Or do we need to first adjust airflow rates for defect ratios in `apply_hvac_finalize_airflows`?

Reply:
Yes, the maximum airflow rate should be applied in a very similar way that fixed capacities are handled. I think the new  `apply_hvac_maximum_airflows` should be called immediately prior to `apply_hvac_fixed_capacities`. This will ensure the entered fixed capacity (if provided) will take priority. However, I wonder if the airflow degradation factor for the installation quality inputs (used in `apply_hvac_finalize_airflows`) need to be considered before adjusting for the maximum allowed airflow.